src/server/main.go

```diff
@@ -2,7 +2,6 @@ package main
 
 import (
 	"context"
-	"flag"
```

Comment:
Not mandatory, but it would be cool to have a `flag`-like interface that reads from env vars instead so we keep types.

Reply:
Other than `bool` or `string`, are there other types that will need to be considered in the future? For now, the change I am implementing will handle only those two, but can be expanded on if need be.

Reply:
`int` and `uint` if you're up for it.  We probably don't care about differentiating between the different sizes (like `uint16` and `uint32`)